docs/server/settings.rst

```diff
@@ -273,6 +273,14 @@ Configuration settings for applications used by Pootle.
   Enable spam prevention through a captcha.
 
 
+.. setting:: POOTLE_REPORTS_MARK_FUNC
+
+``POOTLE_REPORTS_MARK_FUNC``
+  Default: ''
+
+  Provide mark data for user daily activity charts in admin Pootle reports
```

Comment:
I think you need to expand on this, otherwise no one has a clue what this function is supposed to look like, what it should return, and what will the outcome be.